UIKitProjects/BirthdayViewController.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  BirthdayViewController.swift
+//  UIKitProjects
+//
+//  Created by Pavel on 25.09.22.
+//
+
+import UIKit
+/// Контроллер который показывает список персон с их днями рождения
+final class BirthdayViewController: UIViewController {
+    
+    lazy var views: [UIView] = [
+        // First
+        makeDefaultAvatar(yCoord: 115),
+        makeNameLabel(text: "Jeremy", yCoord: 110),
+        makeDaysCount(text: "18 дней", yCoord: 110),
+        makeDescription(text: "10 марта, в среду исполнится 25 лет", yCoord: 135),
+        // Second
+        makeDefaultAvatar(yCoord: 185),
+        makeNameLabel(text: "Maria Lui", yCoord: 170),
+        makeDaysCount(text: "28 дней", yCoord: 170),
+        makeDescription(text: "30 марта, в четверг исполнится 20 лет", yCoord: 205),
+        // Third
+        makeDefaultAvatar(yCoord: 255),
+        makeNameLabel(text: "Jony Stark", yCoord: 240),
+        makeDaysCount(text: "18 дней", yCoord: 240),
+        makeDescription(text: "20 апреля, в субботу исполнится 25 лет", yCoord: 275),
+    ]
+        
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupView()
+    }
+    
+    private func setupView() {
+        title = "Birthday"
+        navigationItem.rightBarButtonItem = UIBarButtonItem(
+            barButtonSystemItem: .add,
+            target: self,
+            action: #selector(addAction)
+        )
+
+        for subview in views {
+            view.addSubview(subview)
+        }
+    }
+
+    @objc func addAction() {
```

Comment:
```suggestion
    @objc func addAction() {
```
располагаться должны выше чем private func